pha-admin/src/main/java/com/store/pharmacy/Company/controller/CompanyController.java

```diff
@@ -0,0 +1,32 @@
+package com.store.pharmacy.Company.controller;
+
+import com.store.pharmacy.Company.model.CompanyInput;
+import com.store.pharmacy.Company.model.CompanyOutput;
+import com.store.pharmacy.Company.service.CompanyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/companies")
+public class CompanyController {
+
+    @Autowired
```

Comment:
A ClientController is using @RequiredArgsConstructor instead of @Autowired and a CompanyController is using @Autowired instead of @RequiredArgsConstructor. Please only use either way.